### PR TITLE
Added vector tiles, which now shows the map at lower zoom level to see the whole UK

### PR DESCRIPTION
**Pull request by jrajak10**
#144 
#146

added a couple of HTML/CSS/JS changes to maintain the styles too.

### CODE REVIEW

File: docs/geovation-community-map/community_map.html [hunk 1]
```diff
@@ -15,37 +15,37 @@
...
-            <hr>
+            <div id="line"></div>
             <div id="startups-supported-legend" class="legend">
                 <h4>Startups Supported</h4>
                 <div class='values'>
-                    <div><span class="large-startups-supported-values"></span>More than 3</div>
-                    <div><span class="medium-startups-supported-values"></span>2-3</div>
-                    <div><span class="small-startups-supported-values"></span>1</div>
-                    <div><span class="zero-startups-supported-values"></span>0</div>
+                    <div><span id="color" class="large-startups-supported-values"></span>More than 3</div>
```
[the remark this answers is not in the document]
Comment (jrajak10) [reply docs/geovation-community-map/community_map.html:27]: The color for the keys on the legend. I wanted to add opacity so it's the same colour as the map

File: docs/geovation-community-map/community_map.css [hunk 2]
```diff
@@ -128,8 +132,11 @@ h4 {
     padding-top: 10px;
 }
 
-hr{
-    border-color: #000;
+#line{
```
[the remark this answers is not in the document]
Comment (jrajak10) [reply docs/geovation-community-map/community_map.css:135]: I'll change it to class. The hr element kept disappearing so I changed it to a div and it works

[the remark this answers is not in the document]
Comment (jrajak10) [reply docs/geovation-community-map/community_map.css]: https://github.com/Geovation/sandbox/pull/147/commits/517d6bac53d6c0b059d19334483964d6ad23b9af

File: docs/geovation-community-map/community_map_functions.js [hunk 3]
```diff
@@ -218,28 +218,50 @@ function toggleLegend(currentLegend, inactiveLegend1, inactiveLegend2) {
...
+        responsiveDisplay('map-information');
     });
 }
 
-function toggleOptions(options, show, hide){
+
+
+function hideLayer(map, hideLayer, inactiveLayer1, inactiveLayer2, inactiveLayer3) {
+    document.getElementById(hideLayer).addEventListener('click', function () {
+        map.setLayoutProperty(inactiveLayer1, 'visibility', 'none');
```
[the remark this answers is not in the document]
Comment (jrajak10) [reply docs/geovation-community-map/community_map_functions.js:245]: That's to ensure the different layers are turned off, and you cans see the map

[the remark this answers is not in the document]
Comment (sebastianovide) [reply docs/geovation-community-map/community_map_functions.js:245]: a lot of repetition.... any other cleaner way ?

File: docs/geovation-community-map/community_map.html [hunk 4]
```diff
@@ -15,37 +15,37 @@
...
+            <div class="line"></div>
             <div id="startups-supported-legend" class="legend">
                 <h4>Startups Supported</h4>
                 <div class='values'>
-                    <div><span class="large-startups-supported-values"></span>More than 3</div>
-                    <div><span class="medium-startups-supported-values"></span>2-3</div>
-                    <div><span class="small-startups-supported-values"></span>1</div>
-                    <div><span class="zero-startups-supported-values"></span>0</div>
+                    <div><span id="color" class="large-startups-supported-values"></span>More than 3</div>
+                    <div><span id="color" class="medium-startups-supported-values"></span>2-3</div>
```
Comment (sebastianovide) [inline docs/geovation-community-map/community_map.html:28]: https://www.w3schools.com/html/html_id.asp

Comment (jrajak10) [reply docs/geovation-community-map/community_map.html]: https://github.com/Geovation/sandbox/pull/147/commits/c7fd5c12c1e06283c165a38933a55131545452b7

File: docs/geovation-community-map/community_map_functions.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (jrajak10) [reply docs/geovation-community-map/community_map_functions.js:245]: There isn't unfortunately.

**Pull request closed without merging**